security/remote_devices/docker-swarm/create-volumes.sh

```diff
@@ -1,31 +0,0 @@
-#!/bin/bash
```

Comment:
Just created an edgex-secrets volume that contains everything for simplicity's sake.